
# Header is in imu_data
sensor_msgs/Imu imu_data

geometry_msgs/Quaternion gt_orientation
geometry_msgs/Vector3 gt_position
geometry_msgs/Vector3 gt_angular_velocity
geometry_msgs/Vector3 gt_linear_acceleration
